tests/cases/fourslash/autoImportSameNameDefaultExported.ts

```diff
@@ -0,0 +1,35 @@
+/// <reference path="fourslash.ts" />
+
+// @module: commonjs
+
+// @Filename: /node_modules/antd/index.d.ts
+//// declare function Table(): void;
+//// export default Table;
+
+// @Filename: /node_modules/rc-table/index.d.ts
+//// declare function Table(): void;
+//// export default Table;
+
+// @Filename: /index.ts
+//// Table/**/
```

Comment:
to make sure I understand this: without the new code, would only rc-table show up? Or would neither show up?